plugins/perms/perms.plugin.zsh

```diff
@@ -0,0 +1,78 @@
+# Some useful commands for setting permissions.
+#
+# Rory Hardy [GneatGeek]
+# Andrew Janke [apjanke]
+
+### Aliases
+
+# Set all files' permissions to 644 recursively in a directory
+alias set644='find . -type f -print0 | xargs -0 chmod 644'
```

Comment:
You can use zsh globbing to select all files in the current directory and subdirectories: `chmod 644 **/*(.)`